editor.planx.uk/src/@planx/components/Pay/Public/Confirm.tsx

```diff
@@ -43,54 +53,61 @@ const PayText = styled(Box)(({ theme }) => ({
 export default function Confirm(props: Props) {
   const theme = useTheme();
 
-  return (
-    <Box textAlign="left" width="100%">
-      <Container maxWidth="md">
-        <Typography variant="h1" gutterBottom align="left">
-          {props.title}
-        </Typography>
-      </Container>
+  const [page, setPage] = useState<"Pay" | "InviteToPay">("Pay");
+  const [nomineeName, setNomineeName] = useState<string | undefined>();
+  const [nomineeEmail, setNomineeEmail] = useState<string | undefined>();
+  const [validatedNomineeEmail, setValidatedNomineeEmail] = useState<
+    string | undefined
+  >();
+  const [showNomineeEmailError, setShowNomineeEmailError] =
+    useState<boolean>(false);
 
-      <Banner
-        color={{
-          background: theme.palette.primary.main,
-          text: theme.palette.primary.contrastText,
-        }}
-      >
-        <Container maxWidth="md">
-          <Typography
-            variant="h5"
-            gutterBottom
-            className="marginBottom"
-            component="h2"
-          >
-            The planning fee for this application is
-          </Typography>
-          <Typography
-            variant="h1"
-            gutterBottom
-            className="marginBottom"
-            component="span"
-          >
-            {formattedPriceWithCurrencySymbol(props.fee)}
-          </Typography>
-          <Typography variant="h4" component="span">
-            <ReactMarkdownOrHtml source={props.description} openLinksOnNewTab />
-          </Typography>
-        </Container>
-      </Banner>
+  const changePage = () => {
+    if (page === "Pay" && !Boolean(props.paymentStatus)) {
+      setPage("InviteToPay");
+    } else {
+      setPage("Pay");
+    }
+  };
 
+  const handleNomineeInputChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    name: string
+  ) => {
+    const input = e.target.value;
+    if (name === "nomineeName") {
+      setNomineeName(input);
+    } else if (name === "nomineeEmail") {
+      setNomineeEmail(input);
+    }
+  };
+
+  const handleNomineeEmailCheck = () => {
+    if (!nomineeEmail || !emailRegex.test(nomineeEmail)) {
+      setShowNomineeEmailError(true);
+    } else {
+      setValidatedNomineeEmail(nomineeEmail);
+    }
+  };
+
+  const payBody = () => (
+    <>
       {!props.error ? (
         <Card>
           <PayText>
-            <Typography variant="h3">How to pay</Typography>
-            <Typography variant="body2">
-              You can pay for your application by using GOV.UK Pay.
+            <Typography variant="h3">
+              {props.instructionsTitle || "How to pay"}
```

Comment:
this isn't ideal from a content perspective (ideally default copy lives exclusively in `formik.initialValues` in the Editor), but it allows these new Editor changes to merge without having to immediately update all existing Pay components (because they will have undefined `props.instructionsTitle`). 

as Pay components are updated later (they'll have to be when invite to pay is launched), we can come back and clean this up. i'm using this for three props: `props.instructionsTitle`, `props.instructionsDescription` and `props.bannerTitle`